### DISCUSSION TITLE
Anonymous connection

### DISCUSSION THREAD

**Discussion by rosh0450** [emqx/emqx#12693] [Q&A]
How to allow emqx exporter to anonymously connect to emqx4.4 using port 18083 without any auth if they are running in the same pods but as different containers?
and, how to get emqx4.4 prometheus metrics using port 8081 without any auth?

**Comment by ieQu1** [comment]
You can create an internal listener as explained here, https://www.emqx.io/docs/en/v4.4/configuration/configuration.html#listener-tcp-internal 
It will have its own configuration zone: https://www.emqx.io/docs/en/v4.4/configuration/configuration.html#listener-tcp-internal-zone
Zones allow to override certain configuration, including the authorization: https://www.emqx.io/docs/en/v4.4/configuration/configuration.html#zone-internal-bypass-auth-plugins

Hopefully it will help.
